libs/common/frontend/models/src/lib/interfaces/app-environment.interface.ts

```diff
@@ -0,0 +1,14 @@
+/**
```

Comment:
Shouldn't this file be named app config as in the backend model, as it won't describe the environment variables rather than the config

Reply:
I could changed it, however in the frontend angular world you typically refer to it as the environment since it is the interface for (`enviornment.ts` / environment.prod.ts). Would you still request me to change it with the information? 
I can simply change it is not a big deal.

Reply:
Alright didn't knew that :)